### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by buildbro**
I've added the new readme for the authgear-server repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
+- Integration with analytics, CDP, and drip campaigns
+- Enterprise Security: Audit logs, Bruteforce Protection, Rate Limits
+- B2B Enterprise Connections: ADFS, LDAP
+- and more...
+
+## What is Authgear Server?
+Authgear server is an open-source project that contains the code that powers Authgear's authentication-as-a-service solution. It includes the code for the server, AuthUI, the Portal, and Admin API. You can use Authgear Server to set up your own self-hosted instance of Authgear service.
+
+## Features and Component
+Authgear Server contains the following fears and components of Authgear:
```
Comment (louischan-oursky) [inline README.md:20]: s/fears/features?

**Comment by louischan-oursky** [comment]
Tag some relevant people @fungc-io

File: README.md [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+- Authgear server (the actual service)
+- Portal (A GUI tool for managing Authgear projects)
+- AuthUI
```
Comment (fungc-io) [inline README.md:23]: @buildbro can you add description for AuthUI?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,56 @@
...
+
+We've provided detailed instructions on how to set up a self-hosted instance of Authgear here: https://docs.authgear.com/deploy-on-your-cloud/local
+
+[Helm Chart](https://docs.authgear.com/deploy-on-your-cloud/helm) is the recommended way to deploy Authgear on Kubernetes for production usage
+
+## How to contribute
+Please refer to [CONTRIBUTING.md](https://github.com/authgear/authgear-server/blob/main/CONTRIBUTING.md) if you need instructions on contributing to the development of authgear-server.
+
+### Authgear SDK
+In addition to Authgear Server, we provide the following SDKs as standalone projects:
```
Comment (fungc-io) [inline README.md:44]: @buildbro please help add some explanation for what the SDKs do?

**Review by fungc-io — CHANGES_REQUESTED** [review]
thanks @buildbro ,
i've add some comments

File: README.md (hunk 1 above)
Comment (buildbro) [reply README.md]: Thank you for spotting that. Fixed!

File: README.md (hunk 2 above)
Comment (buildbro) [reply README.md]: OK! I've updated the readme.

**Comment by fungc-io** [comment]
@louischan-oursky please help merge the pr 🙏

**Pull request merged**